pages/newsletter.js

```diff
@@ -60,7 +60,7 @@ export default function Newsletter() {
                     </div>
                   </div>
                   <div className="flex flex-col items-start">
-                    <div className="rounded-md bg-primary-low dark:bg-primary-medium p-2 ring-1 ring-primary-medium">
+                    <div className="rounded-md bg-primary-low/25 dark:bg-primary-medium dark:ring-primary-medium-low p-2 ring-1 ring-primary-low/100">
```

Comment:
I feel the border around both icons can be a better darker in light mode

Reply:
This opacity/shade was chosen to replicate the previous color `bg-white/5`. The goal of this PR is to replace the bg-[color] with its closest tailwind.config.js configuration. The comment above leans more towards changing the original design. I hope I understood your request correctly.
![svg](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/5b10b709-1508-412c-a2a2-9e1355908a8f)

Reply:
You have changed the ring colour from `primary-medium` to `primary-low/100` which isn't required . you can see this on line 63 as marked by the comment. same is the case which `bg-primary-low`

Reply:
- This change was made to fix the color adjustments from the first pull request within the same issue. Consider viewing the discussion here: https://github.com/EddieHubCommunity/LinkFree/pull/7782#discussion_r1241006010
- I've updated the opacity/shade again to colors that more closely replicate the original colors.
- My choices for the color choice is again purely to replicate the original (before) colors as close as possible to the colors available in the tailwind.config.js
- Here is how the colors look in my code editor, when comparing the **before** and **after** (hence the choice)
![newletter_icon](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/a5409112-7c46-4251-80b4-e610432dea3b)

Reply:
- This change was made to fix the color adjustments from the first pull request within the same issue. Consider viewing the discussion here: https://github.com/EddieHubCommunity/LinkFree/pull/7782#discussion_r1241006010
- I've updated the opacity/shade again to colors that more closely replicate the original colors.
- My choices for the color choice is again purely to replicate the original (before) colors as close as possible to the colors available in the tailwind.config.js
- Here is how the colors look in my code editor, when comparing the **before** and **after** (hence the choice)
![newsletter_icons](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/2303e3a3-f9b5-4b11-9bbd-b535987a05dd)